plotly/tools.py

```diff
@@ -8,21 +8,20 @@
 
 """
 from __future__ import absolute_import
+from collections import OrderedDict
 
 import os.path
 import warnings
 
 import six
-
 import math
 
-
 from plotly import utils
 from plotly import exceptions
 from plotly import session
 
 from plotly.graph_objs import graph_objs
-from plotly.graph_objs import Scatter, Marker
+from plotly.graph_objs import Scatter, Marker, Line, Data
```

Comment:
:cow2: these can be imported on the same line:

`from plotly.graph_objs import Data, Line, Marker, Scatter, graph_objs`